### PR TITLE
7.0 HR fix pep8

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Wrong branch? You need to rebase

File: hr_job_categories/hr.py [hunk 1]
```diff
@@ -108,11 +108,11 @@ def write(self, cr, uid, ids, vals, context=None):
         for contract in self.browse(cr, uid, ids, context=context):
             for data in prev_data:
                 if (
-                        data.get('id') == contract.id
-                        and not vals.get('job_id', False)
-                        or (
-                            data.get('job_id')
-                            and data['job_id'][0] != vals.get('job_id')
+                        data.get('id') == contract.id and
```
Comment (lmignon) [inline hr_job_categories/hr.py]: Do we really need spaces between the '(' and the first checked condition ?
IMO the readability can be improved if the code is written as:

``` python
            if(data.get('id') == contract.id and
               not vals.get('job_id', False) or
               (data.get('job_id') and
                data['job_id'][0] != vals.get('job_id')
               )
            ):
```

or even 

``` python
            if(data.get('id') == contract.id and
               not vals.get('job_id', False) or
               (data.get('job_id') and
                data['job_id'][0] != vals.get('job_id'))):
```

File: hr_labour_recruitment/hr_recruitment.py [hunk 2]
```diff
@@ -68,8 +68,9 @@ def create(self, cr, uid, vals, context=None):
                     'max_employees_fuzz',
                 ], context=context
             )
-            if (data.get('state', 'recruit') != 'recruit'
-                    and int(data['no_of_employee']) >=
+            if (
```
Comment (lmignon) [inline hr_labour_recruitment/hr_recruitment.py]: see previous comment

File: hr_labour_recruitment/hr_recruitment.py [hunk 3]
```diff
@@ -143,8 +144,9 @@ def create(self, cr, uid, vals, context=None):
                     'no_of_employee',
                     'state',
                 ], context=context)
-            if (data.get('state', 'recruit') != 'recruit'
-                    and int(data['no_of_employee']) >= (
+            if (
```
Comment (lmignon) [inline hr_labour_recruitment/hr_recruitment.py]: see previous comment

File: hr_payroll_extension/hr_payroll.py [hunk 4]
```diff
@@ -695,8 +702,9 @@ def get_presence_policies(policy_group_id, day, data):
                             rest_days = actual_rest_days
                             break
 
-                if (scheduled_hours == 0
-                        and dtDateTime.weekday() not in rest_days):
+                if (
```
Comment (lmignon) [inline hr_payroll_extension/hr_payroll.py]: see previous comment

**Comment by ghost** [comment]
@lmignon Thanks, i did the changes.

File: hr_activity_on_timesheet/hr_activity.py [hunk 5]
```diff
@@ -48,10 +48,10 @@ def _get_authorized_user_ids(
...
                     for j in contract_jobs
-                    if j.contract_id.date_start <= now
-                    and (
-                        not j.contract_id.date_end
-                        or now <= j.contract_id.date_end
+                    if j.contract_id.date_start <= now and
+                    (
+                        not j.contract_id.date_end or
+                        now <= j.contract_id.date_end
                     )
```
Comment (dreispt) [inline hr_activity_on_timesheet/hr_activity.py]: Just an opinion: do you really need these hanging parenthesis?

Comment (ghost) [inline hr_activity_on_timesheet/hr_activity.py]: Good point, I made the change. Thanks

**Comment by max3903** [comment]
:+1:

**Comment by ghost** [comment]
@pedrobaeza @lmignon  @dreispt @bwrsandman I would need reviews here, this is a PR that fix PEP8.

**Comment by lmignon** [comment]
:+1:

**Comment by pedrobaeza** [comment]
There are some strange things like `if(something..)` or `'string'and` (without spaces) but it seems that Python compiler nor pe8 don't complain about it, so :+1:

**Pull request merged**